src/main/java/frc/robot/subsystems/colour_wheel/commands/FindColour.java

```diff
@@ -0,0 +1,78 @@
+package frc.robot.subsystems.colour_wheel.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.colour_wheel.ColourWheel;
+
+/**
+ * Spin the wheel until you reach the desired color.
+ */
+public class FindColour extends CommandBase {
+    private ColourWheel colourWheel;
+    private String colour;
+    private double power;
+    private String tempColour;
+    private String[] colours;
+    int finalDis;
+    int tempDis;
+    boolean flag = false;
+
+    public FindColour(ColourWheel colourWheel, String colour, double power) {
+        this.colourWheel = colourWheel;
+        this.colour = colour;
+        this.power = power;
+        colours = new String[]{"YELLOW", "BLUE", "GREEN", "RED"};

```

Comment:
it's not necessary, but I would recommend moving the assignment of colours to the initialization (i.e line14), because if you add another constructor (for example, a constructor with default power), you might forget in assign a value to the array.